Sources/Plasma/FeatureLib/pfDXPipeline/plDXPipeline.cpp

```diff
@@ -8871,8 +8871,8 @@ static void IBlendVertBuffer(plSpan* span, hsMatrix44* matrixPalette, int numMat
         src = inlExtract<hsVector3>(src, vec);
 
         // Destination buffers (float4 for SSE alignment)
-        ALIGN(16) float destNorm_buf[] = { 0.f, 0.f, 0.f, 0.f };
-        ALIGN(16) float destPt_buf[] = { 0.f, 0.f, 0.f, 1.f };
+        alignas(16) float destNorm_buf[] = { 0.f, 0.f, 0.f, 0.f };
+        alignas(16) float destPt_buf[] = { 0.f, 0.f, 0.f, 1.f };
```

Comment:
It's building now, and I'm pretty sure this is the right syntax. This work should be double checked though.